recipes/libx265/all/conanfile.py

```diff
@@ -0,0 +1,133 @@
+from conans import CMake, ConanFile, tools
+import os
+import shutil
+
+
+class Libx265Conan(ConanFile):
+    name = "libx265"
+    description = "x265 is the leading H.265 / HEVC encoder software library"
+    topics = ("conan", "libx265", "codec", "video", "H.265")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = " https://bitbucket.org/multicoreware/x265"
+    exports_sources = "CMakeLists.txt"
+    generators = "cmake"
+    license = ("GPL-2.0-only", "commercial")  # https://bitbucket.org/multicoreware/x265/src/default/COPYING
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "bit_depth": [8, 10, 12],
+        "HDR10": [True, False],
+        "SVG_HEVC_encoder": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "bit_depth": 8,
+        "HDR10": False,
+        "SVG_HEVC_encoder": False,
+    }
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        os.rename("x265_{}".format(self.version), self._source_subfolder)
+
+    def _configure_cmake(self):
+        if self._cmake:
+            return self._cmake
+        self._cmake = CMake(self)
+        self._cmake.definitions["ENABLE_SHARED"] = self.options.shared
+        self._cmake.definitions["ENABLE_LIBNUMA"] = False
+        if self.settings.os == "Macos":
+            self._cmake.definitions["CMAKE_SHARED_LINKER_FLAGS"] = "-Wl,-read_only_relocs,suppress"
+        if self.settings.os != "Windows" and not self.options.shared:
+            self._cmake.definitions["CMAKE_POSITION_INDEPENDENT_CODE"] = self.options.fPIC
```

Comment:
This is handled automatically and should be removed

Reply:
Removed the `PIC` related lines.